Helm: run under Restricted pod security policy (#3007)
grafana/mimir: docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

> **Note:** Although this is not strictly necessary, if you want to access Mimir from outside of the Kubernetes cluster, you will need an ingress. This procedure assumes you have an ingress controller set up.

### Security setup

This installation will not succeed if you have enabled the

> [PodSecurityPolicy admission controller](https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy)

Reviewer: ```suggestion
[PodSecurityPolicy](https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) admission controller
```

Author: are you sure? the links are to the admission controller docs

Reviewer: we don't have "admisson controller" as part of the link in the other cases, so it's for consistency only.

Author: it's a different link in other places

Reviewer: I understand it's a different link, my comment is to make the linking consistent, either make "admission controller" part of the clickable link or not. Also maybe we should actually write "Pod Security Policy" as that's the display name, PodSecurityPolicy is the object name on the API

> or if you are enforcing the Restricted policy with [Pod Security Admission](https://v1-24.docs.kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces).

Reviewer: ```suggestion
or if you are enforcing the Restricted policy with [Pod Security](https://v1-24.docs.kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)  admission controller.
```

The reason is that the installation includes a deployment of MinIO. The [minio/minio chart](https://github.com/minio/minio/tree/master/helm/minio)
is not compatible with running under a Restricted policy or the PodSecurityPolicy that the mimir-distributed chart provides.

If you are using the PodSecurityPolicy admission controller, then it is not possible to deploy the mimir-distributed chart with MinIO.
Refer to [Run Grafana Mimir in production using the Helm chart]({{< relref "../../running-production-environment-with-helm" >}}) for instructions on
setting up an external object storage and disable the built-in MinIO deployment with `minio.enabled: false` in the Helm values file.

> If you are using Pod Security Admission, then MinIO and the mimir-distributed chart can successfully deploy under a Baseline policy.

Reviewer: ```suggestion
If you are using the [Pod Security](https://kubernetes.io/docs/concepts/security/pod-security-admission/) admission controller, then MinIO and the mimir-distributed chart can successfully deploy under a with the [baseline](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels) pod security level.
```

> If you are using the [Pod Security](https://kubernetes.io/docs/concepts/security/pod-security-admission/) admission controller, then MinIO and the mimir-distributed chart can successfully deploy under a with the [baseline](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels) pod security level.

Reviewer: ```suggestion
If you are using the [Pod Security](https://kubernetes.io/docs/concepts/security/pod-security-admission/) admission controller, then MinIO and the mimir-distributed chart can successfully deploy under the [baseline](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels) pod security level.
```
